packages/tools/devtools/devtools-view/src/components/ContainerSummaryView.tsx

```diff
@@ -19,6 +19,8 @@ import {
 	useTableColumnSizing_unstable,
 	useTableFeatures,
 } from "@fluentui/react-components";
+// Allow use of unstable API
+// eslint-disable-next-line import/no-internal-modules
 import { InfoLabel } from "@fluentui/react-components/unstable";
```

Comment:
Any particular reason we moved these from the config? We made the explicit decision to allow imports from this specific path, and have taken a `~` dependency on the library to try to ensure we don't get hit with breaking changes here.

Unless there is a specific reason we want to change the pattern, I think I prefer how it was for a couple few reasons:
1. Centralized location for documentation
2. Less eslint-disable cruft
3. For devs making changes / adding new functionality, it prevents them having to be expressly aware that we've made this allowance - they can just import without issue. I'd like to prevent devs seeing the linter error and assuming they can't use `react-components/unstable`.

Reply:
Yes. Commented in the description:
> Change no-internal-modules to use standard lint config, then disable specific other places as needed.
Maybe there is a better pattern to combine rules for `no-internal-modules`. But I wasn't enamored with a general allowance for something called unstable - seemed like that should allowed on a case-by-case basis. I will defer to your preference.

Reply:
I take it the base config already contains exceptions, and this package's config was effectively removing them? I guess I'd still prefer that, and add whatever other exceptions we need to the override.

In this particular case, we are opting into "experimental" features from FluentUI, and that was a conscious decision. I believe most of the experimental features we opted into have been promoted to stable in more recent releases, but I suspect upgrading would be non-trivial and should be tracked separately. At that point we might be able to remove the exception altogether. I'll add something to Team Rohan's backlog for that.

Reply:
Correct. Restored with rule cloning.